Libs/Widgets/ctkComboBox.cpp

```diff
@@ -104,16 +109,6 @@ QSize ctkComboBoxPrivate::recomputeSizeHint(QSize &sh) const
             }
           }
         break;
-      case QComboBox::AdjustToMinimumContentsLength:
```

Comment:
Is it intentional to remove this case ?

Reply:
Would this break something for folks using Qt4 or older version of Qt5 ?

Reply:
Yes it was intentional.  Pretty much `AdjustToMinimumContentsLength` is deprecated in Qt 5.15, but before then and all the way back to at least Qt 4.8.7, this SizeAdjustPolicy has said in the notes to just `Use AdjustToContents or AdjustToContentsOnFirstShow instead.` (See [here](https://doc.qt.io/archives/qt-4.8/qcombobox.html#SizeAdjustPolicy-enum)).  Seems like something deprecated for a long time, just not specifically marked as deprecated.  The AdjustToContents case a few lines above specifically didn't do this set of code, so I wasn't going to copy it into there.